js/modules/k6/crypto/crypto.go

```diff
@@ -278,6 +281,6 @@ func (hasher *Hasher) Digest(outputEncoding string) (interface{}, error) {
 		return &ab, nil
 
 	default:
-		return nil, fmt.Errorf("Invalid output encoding: %s", outputEncoding)
+		return nil, fmt.Errorf("invalid output encoding: %s", outputEncoding)
```

Comment:
This change makes the linter happy, is the capital here required?

Reply:
The linter is unhappy because errors should not be with a capital letter with the idea that they will be composed and not just a thing that will be returned to the user.  But we are in the *second* situation - this will be the message to the user that they will see in this particular case as an exception and in that case IMO as well as @na-- (from all the discussions we have had around other cases like that) it is better for it to be with a capital letter.

I do actually feel that it's time we disable this particular lint error ?

Reply:
Is not possible to delegate some capital operation in a centralized place before returning it to the user? In that way, we could just exec something like `toUpper(s[0])` and preserve the Go way of composing error messages.